src/tools-support/support.markdown

```diff
@@ -11,29 +11,14 @@ redirect_from: /docs-and-resources/support/
 * [Partners](/tools-support/requesting-partner-support.html)
```

Comment:
On line 7, remove the North America and EMEA text- it's now for all data centers.

Reply:
Done